server/routes/api/users.js

```diff
@@ -50,6 +51,13 @@ router.put(
   UserController.resetPasswordUpdate,
 );
 
+router.get(
+  '/users/feeds',
```

Comment:
This route belongs to the articles route, the route is `/articles/feed` not `/users/feeds`. Check the readme. Correct it so I can approve it.

Reply:
Thanks for spotting this, but the GET /article/ route is already occupied. users/feeds is the next most appropriate route for the business

Reply:
alright